Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/FlyoutSamplePage.xaml

```diff
@@ -180,26 +181,22 @@ turpis semper nisi maecenas, nascetur dictumst sed arcu aenean varius dis leo ha
 										<Flyout.FlyoutPresenterStyle>
 											<Style TargetType="FlyoutPresenter">
 
-												<Setter Property="ScrollViewer.HorizontalScrollMode"
-														Value="Disabled" />
-												<Setter Property="ScrollViewer.HorizontalScrollBarVisibility"
-														Value="Disabled" />
-												<Setter Property="Margin"
-														Value="16" />
+												<Setter Property="ScrollViewer.HorizontalScrollMode" Value="Disabled" />
```

Comment:
keep an eye on the formatting ruling of the project, here setter are not part of newline exception.